Sources/TiqrCoreObjC/Classes/AboutViewController.m

```diff
@@ -121,32 +107,7 @@ - (IBAction)done {
 }
 
 - (void)updateVersion {
-    switch (self.versionInfo) {
-        case AppFriendly:
```

Comment:
I wouldn't call it "friendly" as there is hardly anything friendly about it. I would also make it more descriptive as to the context of the usage of the enum, like:

```
ShowSimpleAppInfo
ShowDetailedAppInfo
ShowLibraryInfo
```